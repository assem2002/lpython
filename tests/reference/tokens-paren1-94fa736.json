{
    "basename": "tokens-paren1-94fa736",
    "cmd": "lpython --no-color --show-tokens {infile} -o {outfile}",
    "infile": "tests/tokens/errors/paren1.py",
    "infile_hash": "2daad629476f5c2f192512f8759c633b7d52da75889d137d32a2d562",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "tokens-paren1-94fa736.stderr",
    "stderr_hash": "8596596679a7a8dc30d0716edf502040725d61cb75a8cc420f0cb295",
    "returncode": 1
}