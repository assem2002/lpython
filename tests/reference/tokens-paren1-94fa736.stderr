tokenizer error: Too many nested parentheses
 --> tests/tokens/errors/paren1.py:1:202
  |
1 | -(((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((1)))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))
  |                                                                                                                                                                                                          ^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
